project/Build.scala

```diff
@@ -267,10 +272,15 @@ object Build {
     version := dottyNonBootstrappedVersion,
     scalaVersion := referenceVersion,
     excludeFromIDE := true,
+
+    disableDocSetting,
   )
 
   // Settings used when compiling dotty with a non-bootstrapped dotty
-  lazy val commonBootstrappedSettings = commonDottySettings ++ Seq(
+  lazy val commonBootstrappedSettings = commonBootstrappedSettings0 ++ Seq(
+    disableDocSetting,
+  )
+  lazy val commonBootstrappedSettings0 = commonDottySettings ++ Seq(
```

Comment:
Looks like commonBootstrappedSettings0 isn't used anymore so this change isn't needed.